snscrape/modules/telegram.py

```diff
@@ -143,6 +143,14 @@ def get_items(self):
 				raise snscrape.base.ScraperException(f'Got status code {r.status_code}')
 			soup = bs4.BeautifulSoup(r.text, 'lxml')
 
+    @staticmethod
+    def get_post_text(post) -> str:
+        result = []
+        # Using the features of the BS4 module itself
+        for s in post.stripped_strings:
+            result.append(s)
+        return '\n'.join(result)
```

Comment:
This can be simplified to `'\n'.join(post.stripped_strings)`, but it doesn't do the right thing anyway. It splits out links into separate lines, and it doesn't preserve multiple line breaks. A good test case for both is <https://t.me/telegram/201>. Looks like this might require explicitly replacing the `<br>` tags.